schemas/learningResource.schema.tpl.json

```diff
@@ -34,6 +34,14 @@
             "https://openminds.ebrains.eu/core/QuantitativeValueRange"
         ]
     },
+    "topic": {
+      "type": "string",
+      "_instruction": "Enter the name or a short description of the aspect of the research product that is covered by this tutorial"
+    },
+    "order": {
```

Comment:
not sure but I think this ordering was not meant here or did I misunderstood the feedback from HE ? 
I thought the request was to order tutorials from the same research product according to how they should be executed

Reply:
if it is just a topic based ordering I would like to solve this more general because we have the same issue for research product orderings in projects which is a long standing request from the community. could we discuss this?

Reply:
There are two goals:
- group tutorials by topic (e.g. for https://docs.arbor-sim.org/en/latest/tutorial/index.html we have topics "Cells", "Recipes", "Networks", etc.)
- order tutorials, either globally or within topics

We already have an order property for live paper sections. Happy to discuss...